install_sct

```diff
@@ -620,6 +620,9 @@ if [ "$ACTUAL_PYTHON" != "$EXPECTED_PYTHON"  ]; then
   exit 1
 fi
 
+# Ensure that packaged GLIBC version is up to date (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3927)
+conda install -y -c conda-forge libstdcxx-ng
```

Comment:
Ah oops! The `libstdcxx-ng` is for Linux only, and not [macOS](https://stackoverflow.com/a/68845839).

Reply:
Fixed in [7eaadee](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3929/commits/7eaadeed1b415889fe03cfb846459a5450d18f7f).